lib/v2/screens/sign_up/viewmodels/signup_event.dart

```diff
@@ -67,6 +67,16 @@ class CreateUsernameOnNextTapped extends SignupEvent {
 }
 
 /// Add Phone Number Events
+class OnCreateAccountTapped extends SignupEvent {
+  const OnCreateAccountTapped({
+    this.phoneNumber,
+  });
+
+  final String? phoneNumber;
+
```

Comment:
At what point does this become null? the text property of the text controller always returns a string even if it is empty.

Also I suggest not use named parameters for just one parameter.

Reply:
User can skip phone number

Reply:
I'm adding `emptyOrNull` check to handle empty string.